themes/digital.gov/layouts/partials/core/guides/guide-menu-bar.html

```diff
@@ -0,0 +1,42 @@
+{{- $guide_data := index $.Site.Data.guidenav -}}
+{{ if .Params.guide }}
+  {{- $guide_data = index $.Site.Data.guidenav (.Params.guide) -}}
+{{ end }}
+{{- $current_page := .Page.RelPermalink -}}
+
+
+<div class="dg-guide__menu-bar">
+  <div class="dg-guide__menu-bar-image-container">
+    <a
+      href="{{- "/" | relURL -}}"
+      title="{{- .Site.Title }} Home"
+      aria-label="{{- .Site.Title }} Home"
+    >
+      <img
+        class="dg-guide__menu-bar-image full"
+        src="{{- "/img/digitalgov-logo.svg" | relURL -}}"
+        alt="Digital Gov Logo"
+      />
+      <img
+        class="dg-guide__menu-bar-image mobile"
+        src="{{- "/img/digit-100.svg" | relURL -}}"
+        alt="Digital Gov Mobile Logo"
+      />
```

Comment:
Using `img` also allows us to include alt text and a title, which I figured might be good for accessibility reasons? This is also the pattern that we use for the site header, so figured it would be good to stay consistent...
```
<div class="usa-logo" id="extended-logo">
      <em class="usa-logo__text">
        <a
          href="{{- "/" | relURL -}}"
          title="{{- .Site.Title }} Home"
          aria-label="{{- .Site.Title }} Home"
        >
          <img
            class="logo-black"
            src="{{- "img/digitalgov-logo.svg" | absURL -}}"
            alt="{{ .Site.Title }} Logo"
          />
        </a>
      </em>
      <p class="sub-head display-none">{{- .Site.Params.description -}}</p>
    </div>
```